### ISSUE TITLE
Inconsistency on sequential analysis plots

### ISSUE THREAD

**Issue by DrMarkGS** [jasp-stats/jasp-issues#613]
* JASP version:
0.12 Feb 5 2020 17.09
Windows 10

I have noted an inconsistency in the sequential analysis plots - for example those produced in the correlation analysis there is a line while in the t-test plots it is just data points.

![image](https://user-images.githubusercontent.com/39003008/74541662-f035a880-4f39-11ea-84c5-2e1d375546cf.png)

Cheers
Mark

**Comment by vandenman** [comment]
Good catch! We'll make this consistent across analyses before the next release.
